app.py

```diff
@@ -25,13 +45,17 @@ def inner():
 		)
 
 		for line in iter(proc.stdout.readline,''):
-			time.sleep(1)                           # Don't need this just shows the text streaming
+			time.sleep(1)  # Don't need this just shows the text streaming
```

Comment:
It was interesting watching this. I think, this can block the reqest fo r other users at the same time if the app is not multi-threaded.